theano/gpuarray/c_code/dimshuffle.c

```diff
@@ -0,0 +1 @@
+#section support_code
```

Comment:
We could also go with a "max case" array of like 32 and check that the input array doesn't have more dimensions than that.